lms/djangoapps/commerce/api/v1/tests/test_views.py

```diff
@@ -175,7 +175,7 @@ def test_update(self):
         self.assertIsNone(VerificationDeadline.deadline_for_course(self.course.id))
 
         # Generate the expected data
-        verification_deadline = datetime(year=2020, month=12, day=31, tzinfo=pytz.utc)
+        verification_deadline = datetime(year=2021, month=12, day=31, tzinfo=pytz.utc)
```

Comment:
This would fail again in 2022. One way is to get the current year and add 1 year in it.

Reply:
fixed